### DISCUSSION TITLE
SSO/OAuth support for OPDS

### DISCUSSION THREAD

**Discussion by WirtsLegs** [gotson/komga#1766] [Ideas]
OPDS v2 doesnt seem to have a spec for OAuth though they are [working on it](https://github.com/opds-community/drafts/discussions/82), however it seems there is a spec for how to accomplish OAuth support with OPDS v1 [here](https://drafts.opds.io/authentication-for-opds-1.0.html#3-authentication-flows)

Adding this support would allow disabling local passwords and entirely move to the SSO provider.

**Comment by gotson** [comment]
I already discussed with the OPDS folks, until they get their spec sorted there's nothing i can do.
